holoviews/plotting/mpl/plot.py

```diff
@@ -19,12 +20,27 @@
 from .util import compute_ratios, fix_aspect
 
 
+@contextmanager
+def rc_context(rcparams):
```

Comment:
I don't think it is at all inefficient to inline - decorators are only applied once at definition time.